core/jazz_cloud-logs-streamer/package.json

```diff
@@ -9,10 +9,16 @@
         "test": "mocha"
     },
     "dependencies": {
-        "unicode-byte-truncate": "*"
+        "aws-sdk": "^2.596.0",
```

Comment:
Don't need to add aws-sdk to dependencies as its available as part of lambda execution environment. For unit tests and local debugging, go ahead and add it to devDependencies.